app/services/ui_notifications/report_finished.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+module UINotifications
+  class ReportFinished < UINotifications::Base
+    include Rails.application.routes.url_helpers
+
+    def initialize(composer, job_id)
+      @composer = composer
+      @job_id = job_id
+      @subject = User.current
```

Comment:
We should always call `super` in the initializer, if the parent class has some logic in the constructor, most likely we still want to call that (unless we want to override it). In this case, we should even use that to set the subject to remain dry. So doing this would be better

```ruby
super(User.current)
```